### ISSUE TITLE
Pagination links for 'next' and 'previous' pages should be understandable even when out of context 

### ISSUE THREAD

**Issue by CharlotteDowns** [alphagov/govuk-frontend#3679]
This issue is from May 2023 external accessibility audit report.

**WCAG Reference:**
2.4.9 Link Purpose (Link Only) (Level AAA)
[Understanding Link Purpose (Link Only) ](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)| [How to Meet Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/quickref/#link-purpose-link-only)

**Issue ID:**
DAC_Link_Purpose_Out_Of_Context_02

**URLs:**
[https://govuk-frontend-pr-3503.herokuapp.com/full-page-examples/search](https://govuk-frontend-pr-3503.herokuapp.com/full-page-examples/search)

## Screen shots
<img width="692" alt="Pagination example" src="https://github.com/alphagov/govuk-frontend/assets/56260216/6358f4ec-4acf-4327-8310-29f3669bd9af">

The ‘Next’ and ‘Previous’ links in the pagination do not possess additional link text of ‘page’ or similar to inform users of screen reading assistive technologies of their purpose when viewing the links out of context using the link text alone.

## Current code ref(s)
`#main-content > form > div.govuk-grid-row > div.govuk-grid-column-two-thirds > nav > div.govuk-pagination__prev > a`

## Screen-reader comments
> “Within the pagination section I identified two links which announced to me as ‘Previous’ and ‘Next’. When situated out of context it may not be immediately clear to all screen reader users what the ‘Next’ and ‘Previous’ links related to. 

> Adding the phrase ‘Page’ to the next and previous links will enable all screen reader users to make a quick and easy selection.” 

> “When navigating in context and out of context, I noticed two links ‘previous’ and ‘next’ as I did not know what this referred to. I found this to be confusing because of the two links which are not clear as this would make it difficult for users to navigate in context or out of context may not pick up what the links are. It would be helpful if the links previous and next could be labelled as previous page and next page.”

## Solution
Ensure that links are descriptive enough when navigating out of context, allowing for users of screen reading assistive technologies to discern their purpose and functionality when doing so.
Consider adding the text ‘page’ to the ‘next’ and ‘previous’ pagination links ot clearly identify that that is what they are used for.

**Comment by davidc-gds** [comment]
This issue has been assessed to **not** be a high-severity accessibility concern. It doesn't appear to meet any of our 3 [criteria for a high-severity accessibility concern](https://design-system.service.gov.uk/community/accessibility-strategy/#determining-severity-of-an-accessibility-concern).

- Despite reasonable evidence that it fails a WCAG AAA criterion, there is no evidence that this makes it difficult or impossible for some people to complete certain tasks. This doesn't mean we shouldn't work to improve the experience, though. ❌
- It does not appear to affect the accessibility of essential services or critical infrastructure. ❌
- There is no blocker to individual service teams efficiently resolving it on their own. ❌

As such, this will not be flagged as a high-priority issue.

In the coming weeks we will investigate whether it fails WCAG level AAA criterion: [2.4.9 Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html).

**Comment by 36degrees** [comment]
Even though this is not a high-priority issue, we think this is likely to be a breaking change, so we're going to try to fix it as part of v5.0.
